public/javascript/admin.js

```diff
@@ -81,10 +81,16 @@ function updateUserOverview() {
     tab.innerHTML = '';
 
     for (chat of chats) {
+        let iconTag = "";
+        if (isNaN(parseInt(chat.icon)))
```

Comment:
Can you surround these statements with brackets? Just because it's what we've been doing elsewhere and if we ever added more lines to these statements we could get weird behavior.